### ISSUE TITLE
running_loss definition

### ISSUE THREAD

**Issue by kd97** [joosthub/PyTorchNLPBook#21]
I can't  understand the code
`running_loss += (loss_t - running_loss) / (batch_index + 1)`
I think loss_t means per current batch loss ,loss_t  is less  than running loss (loss_t - running_loss ?)  can anyone explain it what the codes mean ?
